cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/sponge/platform/CloudNetSpongeClientPingEventResponsePlayers.java

```diff
@@ -0,0 +1,55 @@
+package de.dytanic.cloudnet.ext.bridge.sponge.platform;
+
+import de.dytanic.cloudnet.ext.bridge.server.BridgeServerHelper;
+import de.dytanic.cloudnet.wrapper.Wrapper;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.event.server.ClientPingServerEvent;
+import org.spongepowered.api.profile.GameProfile;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class CloudNetSpongeClientPingEventResponsePlayers implements ClientPingServerEvent.Response.Players {
+
+    static @NotNull ClientPingServerEvent.Response.Players fromCloudNet() {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5823302)